docroot/modules/custom/mass_fields/src/EntityAutocompleteMatcher.php

```diff
@@ -66,10 +66,19 @@ public function getMatches($target_type, $selection_handler, $selection_settings
       foreach ($entity_labels as $bundle => $values) {
         foreach ($values as $entity_id => $label) {
           if (!isset($entity_type_labels[$bundle])) {
-            $bundle_info = $handler->entityTypeBundleInfo->getBundleInfo($target_type)[$bundle];
-            if (isset($bundle_info)) {
-              $entity_type_label = $bundle_info['label'];
-              $entity_type_labels[$bundle] = $entity_type_label;
+            if (isset($handler->entityTypeBundleInfo)) {
```

Comment:
This solution is less than ideal, but it does allow both Move Children page and the English version field to be functional again.

These two autocompletes are different in that they use two different views and two different base entities. The Move Children view, which uses taxonomy, has entityTypeBundleInfo available in the handler, but the English version field does not. The Move Children view similarly unable to retrieve bundle information using entityTypeManager.

Reply:
I think this approach is ok here. entityTypeBundleInfo is not available I think we can keep this approach